Plugins/Libs/Endogine/Endogine.Codecs.Photoshop/FourCC.txt

```diff
@@ -1318,7 +1318,7 @@ phEnumPNGFilterNone	PGNo	PNGFilterNone
 phEnumPagePosCentered	PgPC	pagePosCentered
 phKeyPagePosition	PgPs	pagePosition
 phTypePagePosition		
-phEnumPNGFilterPaeth	PGPt	PNGFilterPaeth
+phEnumPNGFilterPath	PGPt	PNGFilterPath
```

Comment:
If this has an upstream, then I probably should send it there instead of here. Just let me know.

Reply:
I don't think we can change this correctly. It could be a bug in the wrapper or in the PSD format itself, which has spelling errors. Let's leave as-is; it's not a parameter we use anyway.